install/kubernetes/cilium/templates/hubble-ui/service.yaml

```
@@ -7,11 +7,14 @@ metadata:
   labels:
     k8s-app: hubble-ui
 spec:
-  type: ClusterIP
+  type: {{ .Values.hubble.ui.service.type | quote }}
   selector:
     k8s-app: hubble-ui
   ports:
     - name: http
       port: 80
       targetPort: 8081
+      {{- with .Values.hubble.ui.service.nodePort }}
+      nodePort: {{ . }}
+      {{- end }}
```

Let's ensure we set `nodePort` only if the service type is `NodePort`. This is consistent with the way it's done for the clustermesh API server service.
```suggestion
      {{- if and (eq "NodePort" .Values.hubble.ui.service.type) .Values.hubble.ui.service.nodePort }}
      nodePort: {{ .Values.hubble.ui.service.nodePort }}
      {{- end }}
```

---

I'm fine with that, but that will prevent someone who, for some reason, would like to use a `LoadBalancer` service with a fixed node port number.